### ISSUE TITLE
Default user??

### ISSUE THREAD

**Issue by darkguy2008** [linagora/Twake#696]
**Describe the bug**
Hi! Thanks for this great app (or so it seems). I've tried to set it up in my own VPS with docker-compose. So far so good, but I don't know what are the default admin credentials?? The documentation isn't very clear either...

**To Reproduce**
1. Install twake per the docs
2. Open the app
3. Look at the docs...
4. No info! :(

**Expected behavior**
A clear and concise explanation of first-time setup!

**Extra bug (!)**
It seems I can't get past the "Create a new account 1/3", every email I input says it's in use.

Basically this app is unusable :(

**Comment by RomaricMourgues** [comment]
Hi, you just need to press “create an account” and create your first user.
There is no concept of system administrator yet.

**Comment by darkguy2008** [comment]
> Hi, you just need to press “create an account” and create your first user. There is no concept of system administrator yet.
> […](#)
> On 18 Dec 2020, at 21:13, DARKGuy (Alemar) ***@***.***> wrote: Describe the bug Hi! Thanks for this great app (or so it seems). I've tried to set it up in my own VPS with docker-compose. So far so good, but I don't know what are the default admin credentials?? The documentation isn't very clear either... To Reproduce Steps to reproduce the behavior: Install twake per the docs Open the app Look at the docs... No info! :( Expected behavior A clear and concise explanation of first-time setup! — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Thanks for your reply. I did try that but as edited in the OP it says the email is in use, and I'm using emails from my custom domain that's never ever been used before. How do I give you info to debug that? I couldn't see anything in the docker-compose logs :(

**Comment by RomaricMourgues** [comment]
There is an error with the message given, this is the username that is used. We do not accept usernames of less than 5 characters as well as “admin” “guest” and similar words.

Is it your issue ?

**Comment by darkguy2008** [comment]
> There is an error with the message given, this is the username that is used. We do not accept usernames of less than 5 characters as well as “admin” “guest” and similar words. Is it your issue ?

At first I thought it was like that. I tried an email with 7 letters plus @mydomain.com and it didn't work either. I also tried test@mydomain.com, etc. Nothing of that you mention is in the documentation, so it's really lacking in that sense. What other words aren't allowed? It would be a good idea to have that in the docs as well.

In any case, I did try with 7 letters plus @mydomain.com and it said it wasn't available. I also think that error message doesn't actually help much, and there **should** be some way of logging in as an admin to an interface or something where I can at least create users manually or something. This feels as if I'm blind, dealing with a black box and that doesn't help at all to build trust in your app.

**Comment by darkguy2008** [comment]
Nevermind, for some reason it's working now, including admin+1 or other emails/usernames.

This is weird, I did try this in my first startup of the app. I didn't restart the docker-compose process until now. I think that maybe, after the first run, users should stop the containers and restart them again in order for this to work.

This can be closed - but again, the docs need more work! :)
